app/views/api/v2/override_values/base.json.rabl

```diff
@@ -1,3 +1,3 @@
 object @override_value
 
-attributes :id, :match, :value, :use_puppet_default
+attributes :id, :match, :value, :hidden_value, :use_puppet_default
```

Comment:
Not sure this is any help, apart from a little bit of consistency with common parameters.  I'd suggest leaving it out so we can avoid it in future?